inc/gpio.h

```diff
@@ -31,31 +31,31 @@ typedef enum {
 
 // For setting the alternate function on the pin
 typedef enum {
-  GPIO_AF_0 = 0,
-  GPIO_AF_1,
-  GPIO_AF_2,
-  GPIO_AF_3,
-  GPIO_AF_4,
-  GPIO_AF_5,
-  GPIO_AF_6,
-  GPIO_AF_7,
-  GPIO_AF_ANALOG,
-} GPIOAF;
+  GPIO_ALTFN_NONE,
```

Comment:
GPIO_ALTFN_NONE and GPIO_ALTFN_0 now have the same value. Are you sure this is what you want to do?